### DISCUSSION TITLE
Setting Row Heights

### DISCUSSION THREAD

**Discussion by dianventer** [KevinVandy/material-react-table#119] [Q&A] [answered]
Good day,

I've tried changing a bunch of MUI props in the muiProps for the table but am unable to reduce the row height.

So far I've tried this:

muiTableBodyRowProps={{
            sx: {
              height: 10,
              padding: "0px",
              margin: "0px",
              backgroundColor: "#00FF00",
            },
          }}

AS well as this:

 muiTableBodyCellProps={{
            // size: "small",
            sx: {
              fontSize: "10px",
              padding: "none",
              margin: "0px",
              height: "5px",
            },
          }}

But am unable to get the rows to be more "dense"
I undestand that the density changes the height as well. 
I would really appreciate it if someone could assit me with this.

Btw - this libraray is awesome!

**Comment by KevinVandy** [comment, marked as answer]
You have some invalid CSS values up there so that's why its not working. padding: none is invalid CSS

Something like this works fine for what you want:

```jsx
return (
    <MaterialReactTable
    columns={columns}
    data={data}
    muiTableBodyRowProps={{
      sx: {
        height: '10px',
      },
    }}
    muiTableBodyCellProps={{
      sx: {
        p: '2px 16px',
      },
    }}
  />
);
```

<img width="676" alt="image" src="https://user-images.githubusercontent.com/28243511/192065180-fe28a473-522c-43ad-ab6c-14510419691b.png">

**Reply by dianventer** [reply]
Thank you for the assitance!
